src/Raven.Server/Documents/TimeSeries/TimeSeriesStorage.cs

```diff
@@ -889,9 +909,13 @@ void AppendEntireSegment()
                     tvb.Add(segment.Ptr, segment.NumberOfBytes);
                     tvb.Add(collectionSlice);
                     tvb.Add(context.GetTransactionMarker());
+                    tvb.Add(tsNameSlice);
```

Comment:
We explicitly didn't want to have it IIRC. The whole indexing was even designed in a way that the name is not available at the storage level?

This will have severe impact on the storage size.

Reply:
Good point. Will try to avoid this.